### PR TITLE
Added information about paths to `system.merges`.

### PR DESCRIPTION
**Pull request by excitoon**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- New Feature

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):

Added information about paths to `system.merges`.

### CODE REVIEW

**Review by alexey-milovidov — COMMENTED** [review]
Looks Ok.

There are no tests, but how we can test it?
Maybe initiate a long (almost infinite) merge of a table that contains function `sleep` in ORDER BY, then select from system.merges, then cancel merges.

**Comment by excitoon** [comment]
It's not Ok

**Comment by excitoon** [comment]
Now it's Ok

File: dbms/tests/integration/test_system_merges/test.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+        def optimize():
+            node1.query("OPTIMIZE TABLE {name}".format(name=name))
+
+        wait = threading.Thread(target=time.sleep, args=(5,))
+        wait.start()
+        t = threading.Thread(target=optimize)
+        t.start()
+
+        time.sleep(1)
```
Comment (alexey-milovidov) [inline dbms/tests/integration/test_system_merges/test.py]: This will make the test flacky.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**